feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -132,6 +162,30 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
           }
         })
     );
+    this.subscribeForDeeplink();
```

Comment:
is this a breaking change?
https://sap.github.io/spartacus-docs/breaking-changes/
> Changing function behavior where the function returns different results. In other words, when changing function behavior, the function should return the same results for the same provided input as it did before the change.

Reply:
just call private routine, why breaking?

Reply:
Added